apps/api/lib/email/daily-cronjob-reminder-email-templates/mentorship-is-weeks-old-so-request-feedback_mentor.mjml

```diff
@@ -1,5 +1,5 @@
 <mjml>
-  <mj-include path="./head.mjml" />
+  <mj-include path="../templates/head.mjml" />
```

Comment:
Why did we add this change (here and a few more files)? The daily reminder emails have been working fine for months, so I would guess this change likely breaks the implementation.

Beware: the relative path here is **not** relative to the location of this file. In our mjml code it is relative to the folder specified, as seen here: https://github.com/talent-connect/connect/blob/e0d5abe8061275ef4039a4cea412ee0bc6d8dc48/apps/api/lib/email/daily-cronjob-reminder-email.js#L27

Reply:
Without this change, all e-mails in `daily-cronojob-reminder-email-templates` folder look in MJML Preview like this: 
![image](https://user-images.githubusercontent.com/51786805/201352051-0ed6b2be-219d-46db-855c-c20ffa6aea40.png)

Also, I didn't receive the `mentorship-is-weeks-old-so-request-feedback_mentee`email for our mentorship with you, for example. Have you?

That's why I assumed, they didn't work and did these changes. Should have asked you first.

Is there a way to test on dev env if these e-mails are working?

Reply:
Yeah, that issue with it breaking in the Preview is known, and it's expected. It's due to the relative path issue.

About our particular mentorship: I'm not entirely sure, but I think our mentorship went "live" before the automated reminder emails was a thing? But we've also had some issues with them from time to time. But I think they're working fine at the moment.

There's an easy way to see if they're being sent: access the career@redi-school.org account and check the folder where a copy of all outgoing email from CON and TP is stored. Do you have access? Anil or I can give it to you.

There's a way to debug it in development environment, but it's definitely not straight-forward, and I'm the only one who's done it.

Reply:
I will revert them to `<mj-include path="./head.mjml" />` then.

I do not have access to [career@redi-school.org](mailto:career@redi-school.org) account, unfortunately.